llama_stack/cli/stack/run.py

```diff
@@ -65,6 +65,13 @@ def _add_arguments(self):
             type=str,
             help="Path to TLS certificate file for HTTPS",
         )
+        self.parser.add_argument(
+            "--image-type",
```

Comment:
How about choosing a clearer name, such as `--runtime-environment`, `--runtime-env`, or `--runtime`? Since a similar option is used in the `build.py`, it might make sense to update it there as well.

Reply:
I am going to change this flag, but keep `build` the same since I think that is out of scope for this PR. I agree though the naming is a bit overloaded